runtime-config.md

```diff
@@ -12,27 +12,28 @@ Presently there are `Prestart`, `Poststart` and `Poststop`.
 Hooks allow one to run code before/after various lifecycle events of the container.
 Hooks MUST be called in the listed order.
 The state of the container is passed to the hooks over stdin, so the hooks could get the information they need to do their work.
+All hooks execute in the host environment (e.g. the same namespace, cgroups, etc. that apply to the host process).
 
 Hook paths are absolute and are executed from the host's filesystem.
 
 ### Pre-start
 
-The pre-start hooks are called after the container process is spawned, but before the user supplied command is executed.
+The pre-start hooks are called [after the container process is spawned, but before the user supplied command is executed](runtime.md#typical-lifecycle).
```

Comment:
On Mon, Oct 12, 2015 at 01:20:20PM -0700, Vish Kannan wrote:
> You are stating runC's behavior. Why do we have to require that
> behavior in the Spec?

This commit is mostly about describing the runC approach so we have a
non-controversial starting point for future evolution (“is that what
runC does” is something that is harder to disagree on than “should
that be how the spec works” ;).

That being said, I *do* think all runtimes will need to execute
runtime-specified code in the container process [1].

[1]: https://github.com/opencontainers/specs/issues/115#issuecomment-147512709